orchagent/orchdaemon.cpp

```diff
@@ -297,7 +306,8 @@ bool OrchDaemon::init()
      * when iterating ConsumerMap. This is ensured implicitly by the order of keys in ordered map.
      * For cases when Orch has to process tables in specific order, like PortsOrch during warm start, it has to override Orch::doTask()
      */
-    m_orchList = { gSwitchOrch, gCrmOrch, gPortsOrch, gBufferOrch, gIntfsOrch, gNeighOrch, gNhgOrch, gRouteOrch, copp_orch, qos_orch, wm_orch, policer_orch, tunnel_decap_orch, sflow_orch, debug_counter_orch, gMacsecOrch};
+
```

Comment:
Please remove this extra line and also move this to line 383 as the end.

Reply:
Done

Reply:
i think you missed this comment

Reply:
Thanks @prsunny. Removed the extra lines.